bufferserver/src/main/java/com/datatorrent/bufferserver/internal/DataList.java

```diff
@@ -821,34 +848,36 @@ protected void release(boolean wait, boolean writer)
       final int refCount = this.refCount.decrementAndGet();
       if (canEvict(refCount, writer)) {
         assert (next != null);
-        final Runnable storer = getStorer(data, readingOffset, writingOffset, storage);
-        if (future != null && future.cancel(false)) {
-          logger.debug("Block {} future is cancelled", this);
-        }
-        final int numberOfInMemBlockPermits = DataList.this.numberOfInMemBlockPermits.get();
-        if (wait && numberOfInMemBlockPermits == 0) {
-          future = null;
-          storer.run();
-        } else if (numberOfInMemBlockPermits < MAX_COUNT_OF_INMEM_BLOCKS / 2) {
-          future = storageExecutor.submit(storer);
+        if (storage != null) {
```

Comment:
```
if (storage == null) {
  resumeSuspendedClients(MAX_COUNT_OF_INMEM_BLOCKS);
  return;
}
```
to avoid code reformat.
Should it call areSubscribersBehindByMax() before resuming suspended clients?